src/software/pipeline/main_cameraInit.cpp

```diff
@@ -830,6 +830,23 @@ int aliceVision_main(int argc, char **argv)
       vitem.second->addMetadata("AliceVision:useWhiteBalance", (useInternalWhiteBalance)?"1":"0");
     }
   }
+
+  //Check unique frame id per rig element
+  {
```

Comment:
I think it makes sense to have this code as a separate function.

Reply:
done